src/pages/CreatedReviewManagePage/index.tsx

```diff
@@ -1,25 +1,71 @@
 //생성한 리뷰 관리 페이지
-import { useSendReview } from '@/apis/hooks'
+import { Suspense } from 'react'
+import { useLocation } from 'react-router-dom'
+import { Header } from '@/components'
+import { useGetReviewQuestion } from '@/apis/hooks'
+import useActiveTab from '@/pages/CreatedReviewManagePage/hooks/useReviewTab'
+import {
+  Tabs,
+  AllResponseReviewByResponser,
+  AllResponseReviewByReceiver,
+} from './components'
+import { PATH } from './constants'
 
 const CreatedReviewManagePage = () => {
-  const { mutate: sendReview } = useSendReview()
+  const { pathname } = useLocation()
+  const reviewId = pathname.replace(`${PATH.REVIEW_MANAGEMENT}`, '')
```

Comment:
이건 replace 보다는
```suggestion
  const reviewId = pathname.split(PATH.REVIEW_MANAGEMENT)[1]
```
이렇게 작성하거나

```suggestion
  const reviewId = pathname.split('/').at(-1) ?? ''
```

이렇게 작성할 수도 있을 것 같아요!